modules/gapi/test/common/gapi_core_tests.hpp

```diff
@@ -123,99 +123,99 @@ struct PrintNormCoreParams
 struct MathOpTest        : public TestWithParamBase<mathOp,bool,double,bool>
 {
     DEFINE_SPECIFIC_PARAMS_4(opType, testWithScalar, scale, doReverseOp);
-    USE_UNIFORM_INIT(MathOpTest);
-};
-struct MulDoubleTest     : public TestWithParamBase<> { USE_UNIFORM_INIT(MulDoubleTest); };
-struct DivTest           : public TestWithParamBase<> { USE_UNIFORM_INIT(DivTest); };
-struct DivCTest          : public TestWithParamBase<> { USE_UNIFORM_INIT(DivCTest); };
-struct MeanTest          : public TestWithParamBase<> { USE_UNIFORM_INIT(MeanTest); };
-struct MaskTest          : public TestWithParamBase<> { USE_UNIFORM_INIT(MaskTest); };
-struct Polar2CartTest    : public TestWithParamBase<> { USE_UNIFORM_INIT(Polar2CartTest); };
-struct Cart2PolarTest    : public TestWithParamBase<> { USE_UNIFORM_INIT(Cart2PolarTest); };
+    USE_UNIFORM_INIT(MathOpTest, false);
+};
+struct MulDoubleTest     : public TestWithParamBase<> { USE_UNIFORM_INIT(MulDoubleTest, true); };
+struct DivTest           : public TestWithParamBase<> { USE_UNIFORM_INIT(DivTest, true); };
+struct DivCTest          : public TestWithParamBase<> { USE_UNIFORM_INIT(DivCTest, true); };
+struct MeanTest          : public TestWithParamBase<> { USE_UNIFORM_INIT(MeanTest, true); };
+struct MaskTest          : public TestWithParamBase<> { USE_UNIFORM_INIT(MaskTest, true); };
+struct Polar2CartTest    : public TestWithParamBase<> { USE_UNIFORM_INIT(Polar2CartTest, false); };
+struct Cart2PolarTest    : public TestWithParamBase<> { USE_UNIFORM_INIT(Cart2PolarTest, false); };
 struct CmpTest           : public TestWithParamBase<CmpTypes,bool>
 {
     DEFINE_SPECIFIC_PARAMS_2(opType, testWithScalar);
-    USE_UNIFORM_INIT(CmpTest);
+    USE_UNIFORM_INIT(CmpTest, false);
```

Comment:
Very invasive

Reply:
@dmatveev you're looking at the old version. please see the latest one